### PR TITLE
Barracuda-waf-iis : Updated README

### PR DESCRIPTION
**Pull request by amalik99**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by amalik99** [comment]
Updated readme to improve content formatting. No Changes to code. Let me know incase of any questions

File: barracuda-waf-iis/README.md [hunk 1]
```diff
@@ -1,52 +1,69 @@
...
 
 Once deployment finishes, you will able to directly access fully configured Barracuda GUI and start publishing your web applications.
 
-##Deployment Solution Architecture 
+## Template Solution Architecture 
 
 This template will deploy: 
 
-- Two storage accounts
+- Two storage account
```
Comment (bmoore-msft) [inline barracuda-waf-iis/README.md]: accounts - s/b plural

File: barracuda-waf-iis/README.md [hunk 2]
```diff
@@ -1,52 +1,69 @@
...
+## Template Solution Architecture 
 
 This template will deploy: 
 
-- Two storage accounts
+- Two storage account 
 -	One Virtual Network with two subnets
 -	One Load Balancer to facilitate RDP access (via NAT Rules) to backend web servers
--	2 Public IP’s, one for Barracuda WAF and other for LB 
+-	2 Public IP’s, one for Barracuda WAF and other for Load Balancer
```
Comment (bmoore-msft) [inline barracuda-waf-iis/README.md]: ... and one for the Load Balancer

File: barracuda-waf-iis/README.md [hunk 3]
```diff
@@ -1,52 +1,69 @@
...
--	User account with admin or contributor access to the given azure subscription
+Azure Subscription with specified payment method (Barracuda WAF is a market place product and requires payment method to be specified in Azure Subscription)
 
-##Deployment Steps  
+## Deployment Steps  
 
 Build your Barracuda WAF environment on Azure in a few simple steps:  
-- Launch the Template by clicking on Deploy on Azure.  
-- Fill in all the required parameter values. Accept the terms and condition on click Purchase. The deployment takes about 30 minutes. 
+- Launch the Template by click on Deploy to Azure button.
```
Comment (bmoore-msft) [inline barracuda-waf-iis/README.md]: Launch the template by clicking on the Deploy to Azure button.

File: barracuda-waf-iis/README.md [hunk 4]
```diff
@@ -1,52 +1,69 @@
...
+Azure Subscription with specified payment method (Barracuda WAF is a market place product and requires payment method to be specified in Azure Subscription)
 
-##Deployment Steps  
+## Deployment Steps  
 
 Build your Barracuda WAF environment on Azure in a few simple steps:  
-- Launch the Template by clicking on Deploy on Azure.  
-- Fill in all the required parameter values. Accept the terms and condition on click Purchase. The deployment takes about 30 minutes. 
+- Launch the Template by click on Deploy to Azure button.  
+- Fill in all the required parameter values. Accept the terms and condition on click Purchase.
```
Comment (bmoore-msft) [inline barracuda-waf-iis/README.md]: Accept the terms and conditions and click Purchase.

**Comment by amalik99** [comment]
Hi Brian, Thanks for review. your comments are addressed,please let me know incase of anything else. Thank you.!

**Pull request merged**